systemservices/marketplace/mesg.yml

```diff
@@ -0,0 +1,414 @@
+      duration:
+        name: ""
+        description: ""
+        type: String
+      from: *inputFrom
+      gas: *inputGas
+      gasPrice: *inputGasPrice
+    outputs:
+      error: *outputError
+      success: *outputTransaction
+  createServiceVersion:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      hash:
+        name: ""
+        description: ""
+        type: String
+      manifest:
+        name: ""
+        description: ""
+        type: String
+      manifestProtocol:
+        name: ""
+        description: ""
+        type: String
+      from: *inputFrom
+      gas: *inputGas
+      gasPrice: *inputGasPrice
+    outputs:
+      error: *outputError
+      success: *outputTransaction
+  disableServiceOffer:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      offerIndex:
+        name: ""
+        description: ""
+        type: String
+      from: *inputFrom
+      gas: *inputGas
+      gasPrice: *inputGasPrice
+    outputs:
+      error: *outputError
+      success: *outputTransaction
+  purchase:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      offerIndex:
+        name: ""
+        description: ""
+        type: String
+      from: *inputFrom
+      gas: *inputGas
+      gasPrice: *inputGasPrice
+    outputs:
+      error: *outputError
+      success: *outputTransaction
+  transferServiceOwnership:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      newOwner:
+        name: ""
+        description: ""
+        type: String
+      from: *inputFrom
+      gas: *inputGas
+      gasPrice: *inputGasPrice
+    outputs:
+      error: *outputError
+      success: *outputTransaction
+  sendSignedTransaction:
+    name: ""
+    description: ""
+    inputs:
+      signedTransaction:
+        name: ""
+        description: ""
+        type: String
+    outputs:
+      error: *outputError
+      success:
+        data:
+          receipt:
+            type: Object
+  isAuthorized:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      from: *inputFrom
+    outputs:
+      error: *outputError
+      success:
+        name: ""
+        description: ""
+        data:
+          authorized:
+            name: ""
+            description: ""
+            type: Boolean
+  checkForDeployment:
+    name: ""
+    description: ""
+    inputs:
+      hash: *hash
+      addresses:
+        name: ""
+        description: ""
+        repeated: true
+        type: String
+    outputs:
+      error: *outputError
+      success:
+        name: ""
+        description: ""
+        data:
+          authorized:
+            name: ""
+            description: ""
+            type: Boolean
+          source:
+            name: ""
+            description: ""
+            type: String
+            optional: true
+          type:
+            name: ""
+            description: ""
+            type: String
+            optional: true
+events:
```

Comment:
Do we listen any events from the core or cli ?
For now not that I know so I wonder if it's not better to add them when we actually need them and keep this service as simple as possible and just add the stuff we actually need

Reply:
The marketplace website backend should listen event and refresh the data accordingly. So yes it's useful.